docs/docs/configuration/auth.md

```diff
@@ -237,7 +237,7 @@ you should define the 'keycloak-group' value to /admin.
     --client-id=<your client's id>
     --client-secret=<your client's secret>
     --redirect-url=https://internal.yourcompany.com/oauth2/callback
-    --oidc-issuer-url=https://<keycloak host>/auth/realms/<your realm>
+    --oidc-issuer-url=https://<keycloak host>/realms/<your realm>
```

Comment:
```suggestion
    # Before Keycloak v17
    --oidc-issuer-url=https://<keycloak host>/auth/realms/<your realm>
    # Keycloak v17+
    --oidc-issuer-url=https://<keycloak host>/realms/<your realm>
```

I would suggest something like this? @JoelSpeed @flobeier and we should definitely add it to v7.4 and v7.5 docs as well.

Reply:
Given that the older Keycloak versions are EOL I find a comment after the example for recent Keycloak versions better. Fits the existing style and copy + paste will be a bit easier this way.